exercises/dominoes/example.tt

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'dominoes'
+
+# Test data version:
+# <%= sha1 %>
+class DominoesTest < Minitest::Test
+  def assert_correct_chain(input_dominoes, output_chain)
```

Comment:
Could you move this to the bottom of the class please.

In real life I'd put it at the top, but here it would be good if the first thing the student saw was the first test they need to implement.